control-plane/config/crd/bases/consul.hashicorp.com_exportedservices.yaml

```diff
@@ -1,13 +1,9 @@
-# Copyright (c) HashiCorp, Inc.
```

Comment:
you have many instances of removing the copyright information due to code generation.

Reply:
Added back manually.